pkg/kv/kvserver/rangefeed/scheduler.go

```diff
@@ -359,18 +401,24 @@ func (ss *schedulerShard) enqueueN(ids []int64, evt processorEventType) int {
 		return 0
 	}
 
+	// For bulk insertions we ignore sampling frequency and just use one sample
+	// per chunk for simplicity.
+	now := timeutil.Now().UnixNano()
```

Comment:
Why not just use the actual sampling frequency, i.e. `i%histogramFrequency`? It's no more code than what's already here.